src/backend/InvenTree/users/serializers.py

```
@@ -230,6 +230,7 @@ class Meta:
             'user',
             'user_detail',
             'in_use',
+            'version',
         ]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)